package.json

```diff
@@ -48,19 +48,17 @@
     "hexo-fs": "^2.0.0",
     "hexo-i18n": "^1.0.0",
     "hexo-log": "^1.0.0",
+    "hexo-renderer-nunjucks": "^1.0.5",
```

Comment:
Plugin repo is temporarily hosted here: https://github.com/oncletom/hexo-renderer-nunjucks

Happy to transfer its ownership—I need either to have "hexojs create repo" permissions or to add an hexojs admin to the repo, and that repo to be transfered to the hexojs org by this person.

Reply:
@NoahDragon Could you help with this? We could add him to `Plugins Team`.